mne/minimum_norm/spatial_resolution.py

```diff
@@ -65,19 +65,14 @@ def resolution_metrics(resmat, src, function='psf', metric='peak_err',
 
     Notes
     -----
-    For details, see [1]_ [2]_.
+    For details, see :footcite:`MolinsEtAl2008` :footcite:`HaukEtAl2019`.
```

Comment:
the footcite directive is documented here: https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html#role-footcite

I think you could do something like this here:

```suggestion
    For details, see `:footcite:`MolinsEtAl2008,HaukEtAl2019`.
```

Reply:
Just saw this, we'll update it on next commit.